arviz/plots/forestplot.py

```diff
@@ -107,7 +107,9 @@ def plot_forest(data, kind='forestplot', model_names=None, var_names=None, combi
                              ncols=ncols,
                              figsize=figsize,
                              gridspec_kw={'width_ratios': width_ratios},
-                             sharey=True)
+                             sharey=True,
+                             constrained_layout=True,
+                             )
```

Comment:
You and @aloctavodia have very different styles on end parentheses and braces :)

Reply:
oy, I don't care one way or another, but know very nice people who do, and sometimes forget which convention I should be working with in which project!  if we can find the right pylint/black/pycodestyle check for that, happy to merge it.

Reply:
The dance of the lonely parenthesis.